packages/teams-js/src/private/interfaces.ts

```diff
@@ -63,6 +63,25 @@ export enum UserSettingTypes {
   theme = 'theme',
 }
 
+/**
+ * @hidden
+ * Hide from docs.
+ * ------
+ * 3P cloud storage scenario changes that can be subscribed to
+ */
+export enum CloudStorageProvider3PScenario {
+  /**
+   * @hidden
+   * Use this key to subscribe to changes in user's 3P cloud storage provider list
```

Comment:
I assume this is for subscribing to when new cloud start providers are added or removed. For the second value (cloudStorageProviderContent) what does that specifically cover?

Reply:
Updated the code comments.
Sharing the same here as well : 
`Use this key to subscribe to cloud storage provider content changes i.e when file(s) is/are added / renamed / deleted / uploaded, the list of content is updated`